pkg/bff/collaborators.go

```diff
@@ -300,6 +348,50 @@ func (s *Server) LoadCollaboratorDetails(collab *models.Collaborator) (err error
 	return nil
 }
 
+// GetAuth0UserInviteURL generates a user ticket in Auth0 to invite the user to the
+// organization and returns the ticket URL. Depending on the current state of the user,
+// this will either be a password change ticket or an email verification ticket. The
+// caller is responsible for sending the user the ticket URL, which is usually done by
+// including it in the user invite email.
+func (s *Server) GetAuth0UserInviteURL(user *management.User, org *models.Organization) (inviteURL *url.URL, err error) {
+	// Ticket should include the redirect to the auth callback and should expire in 7 days
+	redirectURL := fmt.Sprintf("%s?orgid=%s", s.conf.Auth0.RedirectURL, org.Id)
+	expiration := int((time.Hour * 24 * 7).Seconds())
+	ticket := &management.Ticket{
+		UserID:    user.ID,
```

Comment:
Is it possible user could be Nil here? If so you could do a check first and wouldn't need to have a compound If on 366.

Reply:
It is possible, although we've been relying on tests to assert that anything calling this method won't pass in nil values and at the worst case Auth0 gives us back a nil pointer and we go into panic recovery (although unlikely since the Auth0 SDK should have returned an error).

The reason for the nil check on 366 is because the Auth0 SDK declares the [management.User](https://pkg.go.dev/github.com/auth0/go-auth0@v0.7.0/management#User) fields as pointers. So it's not checking that the `user` pointer is nil but rather that the `EmailVerified` field is nil before we try to dereference it.

Reply:
Ok, that makes sense